catalog/app/containers/Bucket/requests.js

```diff
@@ -1009,10 +1024,10 @@ export const getPackageRevisions = withErrorHandling(
         },
         aggs: {
           revisions: {
-            composite: {
-              // the limit is configured in ES cluster settings (search.max_buckets)
-              size: 10000,
-              sources: [{ pointer: { terms: { field: 'key' } } }],
+            terms: {
+              field: 'key',
+              size: 1000000,
```

Comment:
Can you add a comment on this limit? I would also like to document what it means for our users, no more than 1M package instances?